jwst/wfs_combine/tests/test_wfs_combine.py

```diff
@@ -4,18 +4,36 @@
 from jwst.assign_wcs import AssignWcsStep
 from jwst.associations.asn_from_list import asn_from_list
 from jwst.wfs_combine import WfsCombineStep
+from jwst.wfs_combine import wfs_combine
+import numpy as np
 
 
 GOOD = datamodels.dqflags.pixel["GOOD"]
 DO_NOT_USE = datamodels.dqflags.pixel["DO_NOT_USE"]
 SATURATED = datamodels.dqflags.pixel["SATURATED"]
 
 
+def gaussian(x, mu, sig):
+    return 1. / (np.sqrt(2 * np.pi) * sig) * np.exp(-np.power((x - mu) / sig, 2) / 2)
+
+
+def gaussian2d(x, y, mux, muy, sigx, sigy):
+    return gaussian(x, mux, sigx) * gaussian(y, muy, sigy)
+
```

Comment:
astropy.modeling has a Gaussian1D and Gaussian2D

Reply:
I thought that there should be one. I tried using this but it is not a drop-in replacement. If you want, I'll figure out how to make it work.